mutis/tests/test_signal.py

```diff
@@ -1,16 +1,61 @@
 import pytest
 import numpy as np
+from numpy.testing import assert_allclose
 from mutis.signal import Signal
 
 
 @pytest.fixture
-def profile():
-    p = {"times": np.linspace(10, 20, 100)}
-    p["values"] = 0.5 * np.sin(1 * p["times"]) + 0.5 * np.sin(6*p["times"]) + 1
-    return p
+def signal():
+    times = np.linspace(10, 20, 100)
+    values = 0.5 * np.sin(1 * np.linspace(10, 20, 100)) + 0.5 * np.sin(6 * np.linspace(10, 20, 100)) + 1
 
+    return {
+        "fail": Signal(times, values, "test"),
+        "samp": Signal(times, values, "lc_gen_samp"),
+        "psd_nft": Signal(times, values, "lc_gen_psd_nft"),
+        "psd_fft": Signal(times, values, "lc_gen_psd_fft"),
+        "psd_lombscargle": Signal(times, values, "lc_gen_psd_lombscargle"),
+        "ou": Signal(times, values, "lc_gen_ou"),
+    }
 
-def test_generation(profile):
-    signal = Signal(profile["times"], profile["values"], "lc_gen_samp")
+
+@pytest.fixture
```

Comment:
why using a `@pytest.fixture` here?

Reply:
True, I'll remove it

Reply:
Actually, it is needed to define the `ou_param` fixture used in several test functions.